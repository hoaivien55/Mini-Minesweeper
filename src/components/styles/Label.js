import styled from "styled-components";
const Label = styled.label`
	color: #000000b5;
`;

export default Label;
