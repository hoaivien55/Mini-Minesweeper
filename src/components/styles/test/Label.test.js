import renderer from "react-test-renderer";
import Label from "../Label";

it("Label: Virtual DOM snapshot", () => {
	const tree = renderer.create(<Label />).toJSON();
	expect(tree).toMatchSnapshot();
});
