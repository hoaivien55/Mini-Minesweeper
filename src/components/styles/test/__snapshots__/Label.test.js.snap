// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Label: Virtual DOM snapshot 1`] = `
.c0 {
  color: #000000b5;
}

<label
  className="c0"
/>
`;
